programs/bpf_loader/src/syscalls/cpi.rs

```diff
@@ -9,14 +11,64 @@ use {
         },
         transaction_context::BorrowedAccount,
     },
-    std::{mem, ptr, slice},
+    std::{mem, ptr},
 };
 
+fn check_account_info_pointer(
+    vm_addr: u64,
+    expected_vm_addr: u64,
+    field: &str,
+) -> Result<(), Error> {
+    if vm_addr != expected_vm_addr {
+        log::debug!(
```

Comment:
nit: `log::` can be omitted.

Reply:
when we enable the feature, a few programs are going to break because they accidentally move AccountInfo pointers/do weird stuff with AccountInfos. I think at least initially this will make it easier to debug those cases

Reply:
oh, i meant keeping the log itself, just remove the prefix `log::`...

but just noticed this file unusually doing `use log::*`.. ;)

resolving this review conversation now...